src/test/ui/associated-type-projection-from-multiple-supertraits.stderr

```diff
@@ -42,5 +42,7 @@ LL | fn paint<C:BoxCar>(c: C, d: C::Color) {
 
 error: aborting due to 4 previous errors
 
-You've got a few errors: E0191, E0221
-If you want more information on an error, try using "rustc --explain E0191"
+Some errors occurred: E0191, E0221.
+
+For more information about an error, try `rustc --explain E0191`.
+
```

Comment:
Last thing: removing this trailing empty line and the empty line after "Some errors occurred...", then we'll land this with higher priority.